README.md

```diff
@@ -10,51 +10,52 @@ There are two main categories of samples: **camera** and **applications**. The s
 
```

Comment:
I know the paragraph above hasn't been touched in this PR, but I still think it could be better. It seems a bit redundant.
What about this:
```
The samples in the **applications** category demonstrate practical use of the output data from the camera, such as 3D point clouds or 2D images.
```

Reply:
Done - check fixup commit.

Reply:
Looks good.